webapp/components/test-runs-query.html

```diff
@@ -74,6 +74,12 @@
           this.updateQueryParams(this.queryParams);
         }
 
+        // sha is a convenience method for getting (the) single sha.
```

Comment:
I guess all the call sites for this are potentially buggy with PRs, or is that not a problem?

Reply:
Yes, but since we're expecting one SHA in most cases, it's better to have this as a stop-gap for those call sites I've missed (if there are any) that aren't covered by unit tests, than to risk non-pr case breakages.

Reply:
Makes sense. Should the call sites be converted and this eventually removed, or is it fine if it stays?